cif2cell/cif2cell.xml

```diff
@@ -15,23 +15,23 @@
         <requirement type="package" version="2.0.0a3">cif2cell</requirement>
     </requirements>
     <command detect_errors="exit_code"><![CDATA[
-        cif_structure_name=\$(sed 's/ //g' <<< "$cif_structure.name") &&
-        ln -s $cif_structure \$cif_structure_name &&
-        cif2cell -f \$cif_structure_name -p castep -o out.cell
+        file_name=\$(sed 's/ //g' <<< "$file.name") &&
+        ln -s $file \$file_name &&
+        cif2cell -f \$file_name -p castep -o out.cell
     ]]></command>
     <inputs>
-        <param type="data" name="cif_structure" format="cif" label="Structure file to Convert (.cif)" />
+        <param type="data" argument="--file" format="cif" label="Structure file to Convert (.cif)" />
     </inputs>
     <outputs>
-        <data label="Conversion of $cif_structure.name to .cell" name="out_cell" format="cell" from_work_dir="out.cell" />
+        <data label="Conversion of $file.name to .cell" name="out_cell" format="cell" from_work_dir="out.cell" />
```

Comment:
This file name can be changed after the fact, at any time, this label can at some point not be true. I would not change output label, if not really needed.

Reply:
So it would be better to have a label as something static like "Conversion of .cif file to .cell" that doesn't reference the input file's exact name?
Nearly all of our tools access a cheetah variable for the output `label`, so if this isn't the right way to do things it will need to be changed in all the tools, not just this one.

Reply:
Huh, we didn't think about that before. We were trying to create self-descriptive output labels, but yeah that loses its value if someone changes the names of inputs later.

This is worth a review across all tools - now that we've established a convention for tool naming, the default names might work better for us than before. At the very least it sounds like using the immutable dataset numbers is a good idea.